doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md

```diff
@@ -4,37 +4,44 @@ title: 'Type of signature: ftSignatureType'
+| `0x4445000000000010` | _Start transaction_ result                              | No                       | 1.3                    |
+| `0x4445000000000011` | _Finish transaction_ payload                            | No                       | 1.3                    |
+| `0x4445000000000012` | _Finish transaction_ result                             | Only when using **text** | 1.3                    |
+| `0x4445000000000013` | QR code version                                         | No                       | 1.3                    |
+| `0x4445000000000014` | POS serial number                                       | Only when using **text** | 1.3                    |
+| `0x4445000000000015` | TSE process type                                        | Only when using **text** | 1.3                    |
+| `0x4445000000000016` | TSE process data                                        | Only when using **text** | 1.3                    |
+| `0x4445000000000017` | Transaction number                                      | Only when using **text** | 1.3                    |
+| `0x4445000000000018` | Signature counter                                       | Only when using **text** | 1.3                    |
+| `0x4445000000000019` | Start time of the _start transaction_                   | No                       | 1.3                    |
+| `0x444500000000001A` | Log time of the _finish transaction_                    | Only when using **text** | 1.3                    |
+| `0x444500000000001B` | Signature algorithm                                     | Only when using **text** | 1.3                    |
+| `0x444500000000001C` | Log time format                                         | Only when using **text** | 1.3                    |
+| `0x444500000000001D` | Signature                                               | Only when using **text** | 1.3                    |
+| `0x444500000000001E` | Public key of the TSE                                   | Only when using **text** | 1.3                    |
+| `0x444500000000001F` | Process start (first action)                            | Yes*                     | 1.3                    |
+| `0x4445000000000020` | Update-transaction-payload                              | No                       | 1.3.14                 |
+| `0x4445000000000021` | Update-transaction-result                               | No                       | 1.3.16                 |
+| `0x4445000000000022` | Certification identification (e.g. BSI-K-TR-1234-5678). Also contains information about the certification state (for non certified TSEs) and the state of the environmental protection. | Yes                      | 1.3.16                 |
+
+
+\* This value must be printed when using long-running business sequences and/or the implicit flow and **is not included in the QR code**. It is computed with the following formula: `min(cbReceiptMoment, min(chargeItems.Moment), min(payItems.Moment))`
```

Comment:
```suggestion
\* This value must be printed when using long-running business sequences and/or the implicit flow and **must not be included in the QR code**. It is computed with the following formula: `min(cbReceiptMoment, min(chargeItems.Moment), min(payItems.Moment))`
```

Reply:
Actually it is correct as it is 😅 thanks tough